### DISCUSSION TITLE
I might be wrong but please help

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#1013] [(D) Developer]
so like this script (

function closeit(){ prodigydiv.remove()} setTimeout("closeit", 3000);function getallCurrencies() {PIXI.game.prodigy.player.backpack.data.currency=[]x = PIXI.game.state.states.Boot._gameData.currencyfor (i in x) {    PIXI.game.prodigy.player.backpack.data.currency[i] = {"ID": x[i].ID, "N": 9999999999}}}function getallBuddies() {PIXI.game.prodigy.player.backpack.data.follow=[]x = PIXI.game.state.states.Boot._gameData.followfor (i in x) {    PIXI.game.prodigy.player.backpack.data.follow[i] = {"ID": x[i].ID, "N": 1}}}function getallItems() {PIXI.game.prodigy.player.backpack.data.item=[]x = PIXI.game.state.states.Boot._gameData.itemfor (i in x) {    PIXI.game.prodigy.player.backpack.data.item[i] = {"ID": x[i].ID, "N": 9999999999}}}function getallRelics() {PIXI.game.prodigy.player.backpack.data.spellRelic=[]x = PIXI.game.state.states.Boot._gameData.spellRelicfor (i in x) {    PIXI.game.prodigy.player.backpack.data.spellRelic[i] = {"ID": x[i].ID, "N": 99999999999999}}}function getallHats() {PIXI.game.prodigy.player.backpack.data.hat=[]x = PIXI.game.state.states.Boot._gameData.hatfor (i in x) {    PIXI.game.prodigy.player.backpack.data.hat[i] = {"ID": x[i].ID, "N": 9999999999999999}}}function getallOutfits() {PIXI.game.prodigy.player.backpack.data.outfit=[]x = PIXI.game.state.states.Boot._gameData.outfitfor (i in x) {    PIXI.game.prodigy.player.backpack.data.outfit[i] = {"ID": x[i].ID, "N": 99999999999999}}}function getallBoots() {PIXI.game.prodigy.player.backpack.data.boots=[]x = PIXI.game.state.states.Boot._gameData.bootsfor (i in x) {    PIXI.game.prodigy.player.backpack.data.boots[i] = {"ID": x[i].ID, "N": 99999999999999}}}function getallWeapons() {PIXI.game.prodigy.player.backpack.data.weapon=[]x = PIXI.game.state.states.Boot._gameData.weaponfor (i in x) {    PIXI.game.prodigy.player.backpack.data.weapon[i] = {"ID": x[i].ID, "N": 1}}}function getallGems() {PIXI.game.prodigy.player.backpack.data.key=[]x = PIXI.game.state.states.Boot._gameData.keyfor (i in x) {    PIXI.game.prodigy.player.backpack.data.key[i] = {"ID": x[i].ID, "N": 1}}}function FreeMembership(){PIXI.game.prodigy.player.tt=true}  function levelup(){PIXI.game.prodigy.player.data.level=100}function levelUpPets() {    for (let PetLoopTest = 0; PetLoopTest &lt; PIXI.game.prodigy.player.kennel.data.length; PetLoopTest++) {      PIXI.game.prodigy.player.kennel.data[PetLoopTest].level=100    };}var prodigydiv = document.createElement("prodigydiv");prodigydiv.style.width = "100px";prodigydiv.style.height = "100px";prodigydiv.style.background = "black";prodigydiv.style.color = "purple";prodigydiv.innerHTML = '&lt;button type="button" onclick="{prodigydiv.remove();}"&gt;Close&lt;/button&gt; &lt;button type="button" onclick="{FreeMembership();}"&gt;Free Membership&lt;/button&gt; &lt;button type="button" onclick="{PIXI.game.prodigy.debugMisc.smallLoan(10000000000000);}"&gt;1,000,000,000,0000 Coins&lt;/button&gt; &lt;button type="button" onclick="{PIXI.game.prodigy.debugMisc.getAllPets();}"&gt;Get All Pets&lt;/button&gt; &lt;button type="button" onclick="{PIXI.game.prodigy.player.backpack.data.item.push({"N":999999,"ID":130});}"&gt;Infinite Wheel Spins&lt;/button&gt;  &lt;button type="button" onclick="{FreeAtSchool()}"&gt;Unlock Zones at School&lt;/button&gt; &lt;button type="button" onclick="{levelup()}"&gt;Level Up&lt;/button&gt; &lt;button type="button" onclick="{levelUpPets()}"&gt;Level Up Pets&lt;/button&gt; &lt;button type="button" onclick="{PIXI.game.prodigy.player.appearance._name.nickname=7}"&gt;W12-4RD Nickname&lt;/button&gt; &lt;button type="button" onclick="{getallBuddies()}"&gt;Get All Buddies&lt;/button&gt; &lt;button type="button" onclick="{getallItems()}"&gt;Get All Items&lt;/button&gt; &lt;button type="button" onclick="{getallRelics()}"&gt;Get All Relics&lt;/button&gt; &lt;button type="button" onclick="{getallHats()}"&gt;Get All Hats&lt;/button&gt; &lt;button type="button" onclick="{getallOutfits()}"&gt;Get All Outfits&lt;/button&gt; &lt;button type="button" onclick="{getallBoots()}"&gt;Get All Boots&lt;/button&gt; &lt;button type="button" onclick="{getallWeapons()}"&gt;Get All Weapons&lt;/button&gt; &lt;button type="button" onclick="{getallGems()}"&gt;Get All Gems&lt;/button&gt;&lt;p&gt;Custom Prodigy Mod Menu made by R1DD13 #8209&lt;/p&gt;';
document.body.insertBefore(prodigydiv, document.body.firstChild);                               

       Would it work if we replaced al the (pixis) with _.instance

**Comment by ghost** [comment]
im probily wrong but pls help me if im right

**Comment by NX721** [comment]
No.

**Reply by ghost** [reply]
@NX721 why???

**Comment by PatheticMustan** [comment]
A lot of these things were just removed, debugMisc doesn't exist either

**Reply by ghost** [reply]
okay also please no one swear-cuss-curse- I fell that it doesn't give away a friendly look  dose this work???      .assignRandomSpells()

**Reply by ghost** [reply]
Okay I gues im just stupid *crys*

**Reply by PatheticMustan** [reply]
Pet.assignRandomSpells() assigns random spells to a pet. It makes sure that the pet has valid spells that don't crash it
